File: Algorithms/119. Pascal's Triangle II/golang/solution.dp2.go
func getRow(rowIndex int) []int {
	var ans = make([]int, rowIndex+1)
	for i := 0; i < len(ans); i++ {
		ans[i] = 1
	}
	for i := 1; i < rowIndex; i++ {
		for j := i; j > 0; j-- {
			ans[j] += ans[j-1]
		}
	}
	return ans
}

File: Algorithms/119. Pascal's Triangle II/kotlin/solution.dp2.kt
class Solution {
    fun getRow(rowIndex: Int): List<Int> {
        val ans = MutableList<Int>(rowIndex + 1) { 1 }
        for (i in 0..rowIndex) {
            for (j in (i - 1) downTo 1) {
                ans[j] = ans[j] + ans[j - 1]
            }
        }
        return ans
    }
}
